pkg/services/rpcsrv/server.go

```diff
@@ -847,6 +848,9 @@ func (s *Server) calculateNetworkFee(reqParams params.Params) (interface{}, *neo
 			netFee += (int64(na.NKeys) + 1) * s.chain.GetNotaryServiceFeePerKey()
 		}
 	}
+	if len(tx.GetAttributes(transaction.RefundableSystemFeeT)) > 0 {
+		netFee += s.chain.SystemFeeRefundCost()
+	}
```

Comment:
We have deprecated [(c *Client) AddNetworkFee](https://github.com/nspcc-dev/neo-go/blob/9f9c6f5dc0e9e5e8198353ca65b1fdf577bc7ed7/pkg/rpcclient/rpc.go#L1036) method that is about to be removed, but still have to give a proper response. @roman-khimov, do we need to consider its updating?

Reply:
Unless it goes away earlier than this PR gets in.